test/test_yorkinstruments.m

```diff
@@ -0,0 +1,25 @@
+function test_yorkinstruments
+
+% MEM 3gb
+% WALLTIME 00:10:00
+% DEPENDENCY ft_filetype ft_read_header ft_read_data ft_read_event
+
+%%
+% this is where the data is installed on the DCCN compute cluster
+% and where the nightly regression testing is done
+
+filename = dccnpath('/home/common/matlab/fieldtrip/data/test/original/yi.meghdf5');
```

Comment:
please update this to `/home/common/matlab/fieldtrip/data/test/original/meg/yorkinstruments/yi.meghdf5`